common/source/docs/common-non-gps-navigation-landing-page.rst

```diff
@@ -21,6 +21,7 @@ These are the available options that allow a vehicle to estimate its position wi
     OptiTrack motion capture system <common-optitrack>
 [/site]
     Pozyx Beacons <common-pozyx>
+    Nooploop Beacons <common-nooploop>
```

Comment:
to be a little picky, the list is in alphabetical order so it should go above OptiTrack

Reply:
Thank you. Fixed